Jenkinsfile

```diff
@@ -2,12 +2,14 @@
 
 common {
     slackChannel = '#ksqldb-quality-oncall'
+    timeoutHours = 4
```

Comment:
Setting `downStreamValidate = false` drops the default build timeout (from common) from 3 hours to 1 hour which is too short. We should manually set it back to either 3 hours or 4 hours. Going with 4 hours here since we've seen some timeouts at 3 hours lately.